modules/platforms/python/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 #
 
 cmake_minimum_required(VERSION 3.18)
-project(pyignite3 VERSION ${IGNITE_VERSION} LANGUAGES CXX)
+project(pyignite_dbapi VERSION ${IGNITE_VERSION} LANGUAGES CXX)
```

Comment:
Should it be `pyignite3_dbapi`, and `pyignite3` later, to avoid confusion with Ignite 2.x packages?

Reply:
Well, there is no `3` suffix in other packages (nuget, java client), so I decided against it.

Reply:
Good point, agreed.